yarn.lock

```diff
@@ -10379,6 +10344,11 @@ dompurify@^2.0.7:
   resolved "https://registry.npmjs.org/dompurify/-/dompurify-2.0.12.tgz#284a2b041e1c60b8e72d7b4d2fadad36141254ae"
   integrity sha512-Fl8KseK1imyhErHypFPA8qpq9gPzlsJ/EukA6yk9o0gX23p1TzC+rh9LqNg1qvErRTc0UNMYlKxEGSfSh43NDg==
 
+dompurify@^2.1.1:
+  version "2.2.0"
+  resolved "https://registry.npmjs.org/dompurify/-/dompurify-2.2.0.tgz#51d34e76faa38b5d6b4e83a0678530f27fe3965c"
+  integrity sha512-bqFOQ7XRmmozp0VsKdIEe8UwZYxj0yttz7l80GBtBqdVRY48cOpXH2J/CVO7AEkV51qY0EBVXfilec18mdmQ/w==
+
```

Comment:
I'm not an expert here. But why didn't `yarn` collapsed these three dependencies into a single block that resolves to 2.2.0?

Reply:
Because that might change the behavior of existing code, I think, so it takes the safe route until you choose to upgrade it all. It's sometimes frustrating. I have sometimes run https://github.com/atlassian/yarn-deduplicate, but that can also be risky.

Reply:
I didn't mean to block this. I was just curious on why this would happen.

Reply:
Ah no I didn't take it as such, I was just typing that while not at the computer so I wanted to review today instead :)